src/lib/components/forms/ClassDetailsForm.svelte

```diff
@@ -24,9 +24,13 @@
     course: string
     submitting: boolean
     meetingTimes: Date[]
+    datesHeld: Date[]
+    feedbackCompleted: boolean[]
```

Comment:
can you make this a type ClassDetails? same for the other components in this app

Reply:
put in types folder